lib/pymedphys/docs/users/howto/gamma/from-dicom.ipynb

```diff
@@ -4,9 +4,9 @@
    "cell_type": "markdown",
```

Comment:
<p>which slices? why/how were they chosen? DO they depict all of the different orientations? Are these orientations labelled for clarity?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymedphys/pymedphys/pull/1770/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
No, orientation is not labeled on slices. Here slices are on DICOM coordinates. 
For this example, patient orientation is "head first supine" so patient coordinates are DICOM coordinates are the same.

Reply:
Anyway, I would suggest we add an extra example to use DICOM like "feet first supine" 
What do you think?